apps/frontend/core-dapp/src/components/Link.tsx

```diff
@@ -1,25 +1,41 @@
 import NextLink from 'next/link'
-import styled from 'styled-components'
+import styled, { css, SimpleInterpolation } from 'styled-components'
 
-const Anchor = styled.a`
-  :hover {
-    color: ${({ theme }): string => theme.color.primary};
-  }
+const Anchor = styled.a<{ noUnderline?: boolean }>`
```

Comment:
Let's add a `$`. Also, I don't think this prop needs to be nullable since it's internal.

```suggestion
const Anchor = styled.a<{ $noUnderline: boolean }>`
```

Or if you wanna take my suggestion:

```suggestion
const Anchor = styled.a<{ $underline: boolean }>`
```